### PR TITLE
feat: consolidate cosmos form and useFormSend

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Quality of life PR that will make it easier to bring cross-chain changes and not forget Cosmos SDK which was previously its own domain.

- removes cosmos `useFormSend` and uses the common one
- removes cosmos `SendInput`
- removes cosmos `SendFormFields`

Not supposed to be exhaustive consolidation, but removes the main branching/splitting, so we'll eventually be able to remove this branching, and unify the Cosmos send form entirely into the shared form:

https://github.com/shapeshift/web/blob/86d80c07e3c3106a36c010cb998901128df33385/src/components/Modals/Send/Send.tsx#L42-L45

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Sends could be broken on UTXOs, EVM chains, and Cosmos.
Tested ETH/BTC/Cosmos (with memo) sends which all broadcasted with no regressions.

## Testing

- All sends still work
- Memo still works for Cosmos SDK chains
- Fees are still correctly estimated for Cosmos SDK chains
* Ensure all vanity address features work, i.e. sends to `.eth` addresses (ENS), unstoppable domains-supported TLDs, reverse resolution for unstoppable domains, `y.at`s with the flag on.

### Engineering

- Consolidation looks sane

### Operations

- Refer to top-level

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2939** <a href="https://app.graphite.dev/github/pr/shapeshift/web/2939" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2939?utm_source=stack-comment).

File: src/components/Modals/Send/Form.tsx
Comment (gomesalexandre) [inline src/components/Modals/Send/Form.tsx:34]: OCD alphabetical reorder, only Memo has been added

**Review by 0xApotheosis — APPROVED** [review]
A very tidy negative diff PR.

- Tested that UTXO, EVM and CosmosSDK chains still send as expected.
- Memo worked when sending Cosmos.
- Cosmos SDK fee estimate reflected the actual fee.

Consolidation is tight 👌

**Comment by 0xApotheosis** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/2939?utm_source=stack-comment).


Job Reference: 5RTfzUPQ8wDzu6QclVcQ

**Pull request merged**
